src/main/java/seedu/address/logic/commands/patient/EditPatientCommand.java

```diff
@@ -28,7 +28,7 @@
 import seedu.address.model.patient.Temperature;
 
 /**
- * Edits a patient in address book with the selected name.
+ * Edits a patient in the app with the selected name.
```

Comment:
Do you think we should standardize this? Not sure if it is as important in the JavaDocs because it is understandable, but what about the user guide?

Should we refer to the application as `the app`, `the application`, or `Covigent`?

Reply:
Yep, should take note of the standardisation in UG.